test/test-applications/integrations/TestApplication.Wcf.Client.Core/IStatusServiceContract.cs

```diff
@@ -17,7 +17,7 @@
 using System.ServiceModel;
 using System.Threading.Tasks;
 
-namespace TestApplication.Wcf.Shared;
+namespace TestApplication.Wcf.Client.Core;
```

Comment:
In contrib repo there were a change Client.Core -> Client.DotNet.

Any chance to include into this PR?

Ref.https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/857

Reply:
I will create an issue and I will do it in next PR once I have some time.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1874